bundles/org.openhab.binding.hue/README.md

```diff
@@ -13,12 +13,27 @@ There are two types of Hue Bridges, generally referred to as v1 (the rounded ver
 Only noticeable difference between the two generation of bridges is the added support for Apple HomeKit in v2.
 Both bridges are fully supported by this binding.
 
+Both types of bridges are accessed by means of the "CLIP" ('Connected Lighting Interface Protocol') Application Program Interface ('API').
+There are two versions of CLIP - namely CLIP v2 and CLIP v1.
+Philips has stated that any new features (such as dynamic scenes) will only be available on CLIP v2, and in the long term the CLIP v1 API will eventually be removed.
+Nevertheless it is possible that older v1 (round) bridges might not support CLIP v2, so you need to check that the bridge version is at least 1948086000.
+The CLIP v2 API is more modern and has more features, so it is recommended to use CLIP v2 for new openHAB installations.
+
 Almost all available Hue devices are supported by this binding.
 This includes not only the "Friends of Hue", but also products like the LivingWhites adapter.
 Additionally, it is possible to use OSRAM Lightify devices as well as other ZigBee LightLink compatible products, including the IKEA TRÅDFRI lights (when updated). 
 Beside bulbs and luminaires the Hue binding also supports some ZigBee sensors. Currently only Hue specific sensors are tested successfully (Hue Motion Sensor and Hue Dimmer Switch).
 Please note that the devices need to be registered with the Hue Bridge before it is possible for this binding to use them.
 
+### Supported Things (CLIP v2)
+
+The binding supports *bridge*, *light*, *button*, and *sensor* devices.
+Lights can be of any type from a simple on/off light, through dimmable monochrome lights, to full colour dimmable lights.
+Buttons are devices that contain one or more push buttons.
+Sensors can be (for example) light level sensors, temperature sensors, or motion sensors.
```

Comment:
To me this is quite confusing when seeing the chapter below for v1 which mentions a lot of details. Can these two chapters be merged, as I assume the tables below also more or less applies to v2? If there are any differences, this could be added as new columns, i.e. "v2 only".

Reply:
> this is quite confusing 
You need to understand the following..
1. In API v1 the bridge supports a plethora of Thing types where the type of thing is strictly and intimately defined based on its ZigBee device Id (00, 010, 100, 110, 200, 210, 220, 106, 107, 302, 820, 830, 840, 850, etc.). And there are also a plethora of different sub-flavours of devices (Osram, Philips, Ikea, vanilla ZigBee sensors, ZLLPresence, ZLLTemperature, ZLLLightLevel, CLIP Generic status sensor, 4 button dimmer, tap switch, scene controller, etc. etc.

2. By contrast in API v2 they have **completely eliminated all of the above complexity** -- the bridge just supports DEVICEs, and there is just one single Thing type, which AUTOMATICALLY supports the channels appropriate to its capabilities..

> chapter below for v1 which mentions a lot of details
> assume the tables below also more or less applies to v2? 

Due to the complexity described in point 1. above, the respective chapter for v1 is absolutely necessary for the user to create Things and understand them.

By contrast for API v2 the tables, and indeed the WHOLE chapter, are COMPLETELY IRRELLEVANT and TOTALLY UNNECESSARY. In API v2 there is absolutely no need for the user to know anything about the stuff, complexity and blah-blah mentioned in the API v1 chapter.

Philips created API v2 with the specific purpose to eliminate all that stuff, complexity and blah-blah. They created a clean, flat (and dare I say it 'modern') architecture where a device 'just exists' and has a functionality that depends only and utterly on the contents of its RESOURCE JSON, without any need for any superior knowledge by the user.

.. and we should therefore do the same in OH !!

> any differences, this could be added as new columns, i.e. "v2 only".
Absolutely not. Rather the real statement to the user is _"if you are running API v2 please ignore the API v1 chapter below"_..